### ISSUE TITLE
Maya looks: publish paths as an option for look assigner method

### ISSUE THREAD

**Issue by antirotor** [ynput/OpenPype#3327]
## Feature

reference: #1559 

to have looks more flexible, lets publish path along with the ids and let the artist choose when applying look what method of look assignment it should use.

This will bring it closer to usd/alembics and allow to apply look to data coming from different dccs.

[cuID:2f3nc24]

**Comment by BigRoy** [comment]
The paths (or at least the long names) are actually already stored in the look `.json` files currently. For example:
```json
{
  "attributes": [],
  "relationships": {
    "shading_engine_MTLSG": {
      "uuid": "627bbd3a632436a203ff0ef8:a6684026b4a1",
      "members": [{
        "name": "|model_01_:_GRP|model_01_:geo_GRP|model_01_:mesh_a_GES|model_01_:mesh_a_GESShape",
        "uuid": "627bbd3a632436a203ff0ef8:9b7a7cba5128"
      }, {
        "name": "|model_01_:_GRP|model_01_:geo_GRP|model_01_:mesh_b_GES|model_01_:mesh_b_GESShape",
        "uuid": "627bbd3a632436a203ff0ef8:f410f33e62ac"
      }]
    },
  }
}
```

It's currently a `json.dump` of [the "relationships" collected here in CollectLook](https://github.com/pypeclub/OpenPype/blob/2b36eea3258d4d1ba69d1d1a0a5864037fd89f24/openpype/hosts/maya/plugins/publish/collect_look.py#L417-L429).

**Comment by antirotor** [comment]
Yes, I think the easiest way would be to drop namespaces on path, treat the root as wildcard (or drop it off completely) and then try to match the path on selected hierarchy.

**Comment by BigRoy** [comment]
Is this closed in favor of another issue? Since I believe the implementation for path based look assignments isn't implemented?

**Comment by tokejepsen** [comment]
Not sure why the ynbot closer this

**Comment by BigRoy** [comment]
@iLLiCiTiT @mkolar why does ynbot keep closing this incorrectly?

**Comment by jakubjezek001** [comment]
> @iLLiCiTiT @mkolar why does ynbot keep closing this incorrectly

The issue was tagged with false ClickUo ID pointing to different issue which had been already closed. This issue should be now kept since it had been reassigned to new ClickUp task. Thanks @BigRoy for pointing at it!
